src/resources/Extensions/Extensions.ts

```diff
@@ -32,4 +32,13 @@ export default class Extension extends Resource {
     list() {
         return this.api.get<ExtensionModel[]>(Extension.baseUrl);
     }
+
+    /**
+     * Compiles the extension's script
+     * 
+     * @param {ExtensionCompileCode} extensionCode The code to compile
+     */
+    compileCode(extensionCode: ExtensionCompileCode) {
```

Comment:
Is it really compiling the code or it would make more sense to call this `validateCode`?